utils/index.d.ts

```diff
@@ -1496,6 +1506,19 @@ export declare abstract class AzureWizardPromptStep<T extends IActionContext> {
      */
     public configureBeforePrompt?(wizardContext: T): void | Promise<void>;
 
+    /**
+     * Can be optionally added to a step so the properties can be populated in the confirmation web view
+     * @returns An object with three properties
+     *      name: A displayable name of the step
+     *      value: The label of the chosen value for the step
+     *      valueInContext: The name which can be used to access the value in the wizard context
+     */
+    public confirmationViewProperty?(wizardContext: T): {
+        name: string;
+        value: string;
+        valueInContext: string;
+    };
```

Comment:
Since we're using this in multiple places, have you considered making a `ConfirmationViewProperty` interface/type?